ARGUS/ARGUS_Transforms.py

```diff
@@ -136,16 +130,35 @@ def make_slices(smin, smax, _start, _end):
                 self._roi_center_slice - self.boundary,
                 self._roi_center_slice - self.boundary + self.num_slices,
                 _start, _end)
-        img = img[tuple(slices)]
+
+        arr = img[tuple(slices)]
 
         if self.reduce_to_statistics:
+            num_stats = 2
+            if self.extended:

```

Comment:
Would it be ok to change `extended` to `extended_std` given that we get the `stdmin` and `stdmax` frames out of that code block? That would be more specific to what frames we are getting when we enable that parameter.

Reply:
It would break backward compatibility (we will need to fix things in the PTX dir), but I'm fine with fixing the name.  It bothers me too!  :)

How about being more explicit: include_windowed_stddev?  include_windowed_statistics?  include_windowed?

Reply:
I like the parameter names that you suggested. 

Thinking about backward compatibility - we might need to fix all the scripts related to PTX, PNB and ONSD that have been commited so far. This is also a good chance to create those config files and scripts for common functions.